humanoid_league_team_communication/scripts/TeamCommTest.py

```diff
@@ -1,45 +1,43 @@
 #!/usr/bin/env python3
-
 """
 This script publishes dummy values for ball, goalpost, position and obstacles for testing the team communication.
 """
 
 import rclpy
-from rclpy.node import Node
 from geometry_msgs.msg import PoseWithCovariance
 from humanoid_league_msgs.msg import ObstacleRelativeArray, ObstacleRelative, PoseWithCertaintyArray, PoseWithCertainty
 
-
 if __name__ == '__main__':
     rclpy.init(args=None)
-    ball_pub = self.create_publisher(PoseWithCertaintyArray, "balls_relative", 1)
-    position_pub = self.create_publisher(PoseWithCertainty, "pose_with_certainty", 1)
-    obstacle_pub = self.create_publisher(ObstacleRelativeArray, "obstacles_relative", 1)
+    node = rclpy.create_node("team_comm")
```

Comment:
Is this correct? Should the node have its own name e.g. `team_comm_test`?
Also it is also possible to create publishers [directly](https://docs.ros2.org/latest/api/rclpy/api/node.html?highlight=publisher#rclpy.node.Node.create_publisher), but then a QoS profile needs to be set.

Reply:
The name `team_comm_test` is better. The other name is conflicting with the original team comm.

Reply:
> Also it is also possible to create publishers directly, but then a QoS profile needs to be set.

I would do it as is, this is the common way.

Reply:
+1